app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -129,6 +156,29 @@ class DeadlineDetailsActivity : AppCompatActivity() {
         editMode = editMode.not()
     }
 
+    fun saveNotificationSetting(view: View) {
+        val newDeadline = Deadline(titleView.text.toString(), state, dateTime)
+        DeadlineNotification.editNotification(id, newDeadline, retrieveNotificationsTimes(), this)
+    }
+
+    private fun setCheckBoxTexts(){
+        findViewById<CheckBox>(R.id.radio_notification_1h).text =
+            getString(R.string.notify_before, "1 hour")
+        findViewById<CheckBox>(R.id.radio_notification_5h).text =
+            getString(R.string.notify_before, "5 hours")
+        findViewById<CheckBox>(R.id.radio_notification_1d).text =
+            getString(R.string.notify_before, "1 day")
+        findViewById<CheckBox>(R.id.radio_notification_3d).text =
+            getString(R.string.notify_before, "3 days")
+    }
+    private fun setNotificationCheckBox(){
+        val notifications = DeadlineNotification.listDeadlineNotification(id, this)
+        if (notifications.contains(checkBoxIdTime[R.id.radio_notification_1h])) findViewById<CheckBox>(R.id.radio_notification_1h).isChecked = true
+        if (notifications.contains(checkBoxIdTime[R.id.radio_notification_5h])) findViewById<CheckBox>(R.id.radio_notification_5h).isChecked = true
+        if (notifications.contains(checkBoxIdTime[R.id.radio_notification_1d])) findViewById<CheckBox>(R.id.radio_notification_1d).isChecked = true
+        if (notifications.contains(checkBoxIdTime[R.id.radio_notification_3d])) findViewById<CheckBox>(R.id.radio_notification_3d).isChecked = true
+    }
```

Comment:
It will be more readable if you create a function to avoid this code duplication.

Reply:
It's done.